java-bootcamp-bahia-t1/src/CredictCardPayment.java

```diff
@@ -0,0 +1,68 @@
+/*A concrete form to pay ; with Credit Card*/
+
+public class CredictCardPayment implements Payment {
+
+	private String name;
+	private String cardNumber;
+	private String cvv;
+	private String dateOfExpiry;
+	private Discount discount;
+
+	public CredictCardPayment(String name, String cardNumber, String cvv, String dateOfExpiry) {

```

Comment:
The modelling is very accurate with the reality but keep in mind that the enunciate said "by Credit Card: where a Name and a Credit Number is required" (it is just a comment you don't need to change nothing here)